regression/contracts/trivial_contract_enforce/test.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+main.c
+--enforce-all-contracts
+^EXIT=10$
+^SIGNAL=0$
+^\[main.assertion.\d+\] line \d+ assertion foo\(\&n\) != 15\: FAILURE$
+^VERIFICATION FAILED$
+--
+--
+Check whether CBMC doesn't crash when enforcing incomplete contracts, i.e.,
+without requires using ensures and vice-versa.
```

Comment:
I would reword this description a bit:
- In the test, we are not checking the other direction: "ensures without requires" but we say "vice-versa"
- The crash was not strictly due to "incomplete" contracts. It could be triggered by writing `__CPROVER_ensures(true)` as opposed to removing it entirely.

I would say exactly what the PR title says and define what "trivial" means in an enforce context, which is that the postcondition is true (default when missing) and therefore there is nothing to check/assert.

Reply:
Implemented as suggested.